micro-rdk-nmea/src/messages/pgns.rs

```diff
@@ -106,3 +115,96 @@ pub struct GnssSatsInView {
     #[length_field = "sats_in_view"]
     satellites: Vec<Satellite>,
 }
+
+macro_rules! define_pgns {
+    ( $(($pgndef:ident, $enum:ident, $pgn:expr)),* ) => {
+        #[derive(Clone, Debug)]
+        pub enum MessageData {
```

Comment:
Maybe `NmeaMessage` instead of `MessageData`, which is pretty generic.

Reply:
Ah, you have `NmeaMessage` later, which is this + metadata. Maybe this should be `NmeaMessageBody` or something.